apps/debug-reindexing-endpoint/go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/semi-technologies/weaviate-chaos-engineering-reindexing
+module reindexing-endpoint
 
-go 1.22
```

Comment:
we don't need this change anymore if the golang version installed in the system is newer than 1.23, you can leave it as it was.

Reply:
Do you want me to revert this change?

Reply:
As you will be squashing all the commits, simply leave the `go 1.22` instead of `go 1.22.0`, the package naming change is still legit.